### PR TITLE
Push docker image to docker.io for release

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: bin/push-docker.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+!/bin/bash
```
Comment (lookuptable) [inline bin/push-docker.sh]: Is this change intentional?

File: bin/push-docker.sh [hunk 2]
```diff
@@ -16,34 +16,44 @@ function usage() {
   exit 1
 }
 
+function docker_push() {
+  local im="${1}"
+  if [[ "${im}" =~ ^gcr\.io ]]; then
+    gcloud docker -- push "${im}"
+  else
+    docker push ${im}
```
Comment (lookuptable) [inline bin/push-docker.sh]: nit: `${im}` should be doubly quoted.

**Comment by sebastienvas** [comment]
Nice catch.

File: bin/push-docker.sh (hunk 1 above)
Comment (sebastienvas) [inline bin/push-docker.sh]: Done

File: bin/push-docker.sh (hunk 2 above)
Comment (sebastienvas) [inline bin/push-docker.sh]: Done
